paper/paper.Rmd

```diff
@@ -18,7 +18,7 @@ authors:
   orcid: 0000-0001-9560-6336
 - affiliation: 5
   name: Etienne Bacher
-  # orcid: 
+  orcid: 0000-0002-9271-5075 
```

Comment:
Can you please also add this to `DESCRIPTION` file when you make a PR for #188

Reply:
yes I will